details.html

```diff
@@ -152,434 +156,764 @@
-                        watch_later
-                      </span>
+                  <div class="text-info">
+                    <span class="title text-ellipsis">App Development</span>
 
-                      <span>1 week left</span>
-                    </div>
-                  </footer>
-                </div>
-              </li>
+                    <small class="subtitle text-ellipsis">Facebook, Inc.</small>
+                  </div>
+                </a>
+
+                <footer class="footer">
+                  <div>
+                    <img
+                      src="./images/user1.png"
+                      class="avatar round small"
+                      alt="Bashar Asadov avatar"
+                    />
+
+                    <img
+                      src="./images/user2.png"
+                      class="avatar round small"
+                      alt="Valijon Putinov avatar"
+                    />
 
-              <li>
-                <div class="related-projects-list-item">
-                  <a href="#" class="mini-profile">
                     <img
-                      src="./images/angular.jpg"
-                      alt="A red shield with a letter A on it"
-                      class="avatar large"
+                      src="./images/user3.png"
+                      class="avatar round small"
+                      alt="Ibrokhim Alisherov"
                     />
+                  </div>
 
-                    <div class="text-info">
-                      <p class="title text-ellipsis">Admin Dashboard</p>
+                  <div class="badge danger light icon">
+                    <span class="material-icons-outlined icon">
+                      watch_later
+                    </span>
 
-                      <small class="subtitle text-ellipsis">
-                        ArtTemplate, Inc.
-                      </small>
+                    <span>1 day left</span>
+                  </div>
+                </footer>
+              </div>
+            </div>
+          </div>
+
+          <main class="container">
+            <article class="project">
+              <header class="header">
+                <div class="mini-profile">
+                  <img
+                    src="./images/dropbox.jpg"
+                    alt="blue shapes of an open box"
+                    class="avatar large"
+                  />
+
+                  <div class="text-info">
+                    <h1 class="title">App Development</h1>
+                    <small class="subtitle">Dropbox, Inc.</small>
+                  </div>
+                </div>
+
+                <button class="btn icon-only">
+                  <span class="material-icons-round">more_horiz</span>
+                </button>
+              </header>
+
+              <section class="intro">
+                <h2 class="content-title">Details</h2>
+
+                <div class="stats">
+                  <div class="stat-info">
```

Comment:
I believe it had a better name before

Reply:
Don't remember if I said wrong class name for this case before. But even if I did, this new name is worse. Let's discuss it.